### ISSUE TITLE
add a "quicktest" simulation for all the models

### ISSUE THREAD

**Issue by schuelet** [RakipInitiative/ModelRepository#181]

**Comment by llavall** [comment]
This ticket should be assigned with the modeller. I have to adjust the workflows afterwards.

**Comment by llavall** [comment]
The aim here is to assure, that models that run not within a short period of time (lets say around 5 minutes) can be executed in a short time, without the need to have realistic results, just to check if the model parts work as expected.
Therefore, all longer running models need to be investigated, if a change in the model parameters (named quickrun or was it quickest?) will speed up the entire execution process ans this setting then added to the simulations of the model.

**Comment by schuelet** [comment]
Agreed, we should state this in the manuals/wiki if not already done.
The default-sim is the quick self test, any other sims are "real".

**Comment by llavall** [comment]
@schuelet @esther-sundermann @mfilter   If I remember right it was decided a while ago, that we have to change the definition from "The default-sim is the quick self test, any other sims are "real"." to what is written here https://github.com/RakipInitiative/ModelRepository/issues/181#issuecomment-713660776

**Comment by esther-sundermann** [comment]
It was agreed that there is no need to have a quicktest simulation. If the defaultSimulation is fast, than the quicktest is equal to the defaultSimulation.

To ensure downwards compatilitly, default can be a "real" simulation. 

Yes, testing workflows need to be able to read the list of all possible simulations and select the quicktest simulation in the runner via variable, if present, otherwise choose default.

HTH

**Comment by mfilter** [comment]
I changed the title of the ticket. @schuelet: re the point that relates to the test WFs it probably makes sense to have an independent ticket. BTW, in cases where a seperate quicktest simulation is provided in addition to the default simulation it would be good to have this as the first entry in the simulation drop-down list of the landing page. This has however lower prio.
